docs/api.rst

```diff
@@ -4,6 +4,9 @@ Read the Docs Public API
 We have a limited public API that is available for you to get data out of the site. 
 This document covers only part of the API provided. We have plans to create a read/write API, so that you can easily automate interactions with your project.
 
+.. warning:: This API is out of date and not currently maintained.
+             We have a v2 API that is in better repair available at http://readthedocs.org/api/v2/.
```

Comment:
_is in better repair available_, is that phrase is correct? I'm not sure to understand what it means.